src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentQuickActions/QuickActionButton/QuickActionButton.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import React, { type Element } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { Button, spacers } from '@dhis2/ui';
+import { Tooltip, withStyles } from '@material-ui/core';
+
+const styles = {
+    button: {
+        display: 'flex',
+        gap: spacers.dp8,
+        alignItems: 'center',
+    },
+};
+
+type Props = {|
+    icon: Element<any>,
+    label: string,
+    onClickAction: () => void,
+    dataTest?: string,
+    disable?: ?boolean,
+    ...CssClasses,
+|}
```

Comment:
Can we move this to a separate file? Also, I've found that it is, in most cases, better to use the annotation `disable?: boolean` to not allow null values; the benefit being that we can assign default values in the function parameters if we need it.